modules/lvms-about-adding-devices-to-a-vg.adoc

```diff
@@ -10,6 +10,15 @@ The `deviceSelector` field in the `LVMCluster` CR contains the configuration to
 
 You can specify the device paths in the `deviceSelector.paths` field, the `deviceSelector.optionalPaths` field, or both. If you do not specify the device paths in both the `deviceSelector.paths` field and the `deviceSelector.optionalPaths` field, {lvms} adds the supported unused devices to the volume group (VG). 
 
+[WARNING]
+====
+It is recommended to avoid referencing disks using symbolic naming, such as `/dev/sdX`, as these names may change across reboots within RHCOS. Instead, you must use stable naming schemes, such as `/dev/disk/by-path/` or `/dev/disk/by-id/`, to ensure consistent disk identification.
```

Comment:
I won't stop the merge over this, but I think there's a bit of a mismatch here where the first sentence says it's "recommended" not to do one thing, but the second sentence says you "must" do the other thing instead.

Just something to consider.